Firmware/Marlin_main.cpp

```diff
@@ -4117,62 +4173,52 @@ void process_commands()
         if (farm_prusa_code_seen()) {}
         else if(code_seen_P(PSTR("FANPINTST"))) {
             gcode_PRUSA_BadRAMBoFanTest();
-        }
-        else if (code_seen_P(PSTR("FAN"))) { // PRUSA FAN
+        } else if (code_seen_P(PSTR("FAN"))) { // PRUSA FAN
             printf_P(_N("E0:%d RPM\nPRN0:%d RPM\n"), 60*fan_speed[0], 60*fan_speed[1]);
-        }
-        else if (code_seen_P(PSTR("uvlo"))) // PRUSA uvlo
-        {
-            if (eeprom_read_byte((uint8_t*)EEPROM_UVLO_PRINT_TYPE) == PowerPanic::PRINT_TYPE_SD)
-            {
+        } else if (code_seen_P(PSTR("uvlo"))) { // PRUSA uvlo
+            if (eeprom_read_byte((uint8_t*)EEPROM_UVLO_PRINT_TYPE) == PowerPanic::PRINT_TYPE_SD) {
```

Comment:
Can `saved_printing_type` be used here?

`if (saved_printing_type == PowerPanic::PRINT_TYPE_SD)`

Reply:
When you assume that `PRUSA uvlo` is only used by the "normal" Power Panic recovery functions then the `saved_printing_type` is already read before the `PRUSA uvlo`  is called.
But as this is a gcode that can be executed any time the `saved_printing_type` may not be correct.
I have no idea why this gcode even exists.